source/stylesheets/layout/_header.scss

```diff
@@ -0,0 +1,162 @@
+@import "mediaqueries/media-header";
+@import "components/_button";
+
+// import fontello icons
+@import "icons/fontello";
+@import "icons/fontello-ie7";
+@import "icons/fontello-ie7-codes";
+@import "icons/fontello-codes";
+@import "icons/fontello-embedded";
+
+//services navigation styling
+.services-nav {
+  display: flex;
+  justify-content: space-between;
+  margin: $base-margin 0;
+  p {
+    color: $grey;
+    font-weight: 400;
+    margin: $base-margin;
+  }
+  .phone-number {
+    color: $turquoise;
+  }
+  .register-nav {
+    display: inline-flex;
+    height: 2em;
+    a {
+      font-size: $small-font;
+      font-weight: 400;
+      text-transform: uppercase;
+      color: $grey;
+      margin-top: $base-margin;
+      &:hover, &:focus {
+        text-decoration: none;
+      }
+      &:after {
+        content: "/";
+        font-weight: 900;
+        margin: 0 $base-margin;
+      }
+    }
+    .register {
+      color: $turquoise;
+      text-decoration: none;
+    }  
+  }
+  .language {
+    width: 15px;
+    margin-bottom: $base-margin/2 
+  }
+}
+.register-nav a.no-after:after{
+  content: " ";
+}
+.search-group {
+  width: 140px;
+  height: 22px;
+  margin-right: $base-margin*4;
+  margin-top: $base-margin/2;
+  .form-search{
+    height: 22px;
+    font-size: $small-font;
+    box-shadow: none;
+    border-radius: 7px 0 0 7px ;
+    margin: 0;
+    padding: 8px;
+    &:focus {
+    border-color: $turquoise;
+    outline-color: none;
+    }
+  }
+  .search-button {
+    @include search-button;
+  }
+}
+.text-slider-js div {
```

Comment:
this is only class for jQuery selectionc. It should not have any properties. This should be reduced only to ```js-text-slider``` It should be prefixed on the beginning of the phrase